galleries/examples/text_labels_and_annotations/legend_demo.py

```diff
@@ -13,6 +13,7 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
+import pylab
```

Comment:
Please note that use of pylab is now [strongly discouraged](https://matplotlib.org/stable/api/index.html#module-pylab), so we would not want to use it in our examples.